packages/dashboard-backend/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@eclipse-che/dashboard-backend",
+  "version": "7.33.0-SNAPSHOT",
+  "description": "Dashboard backend for Eclipse Che",
+  "main": "lib/server.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "rimraf lib && webpack-cli --mode development --config webpack.config.js",
+    "prepare": "yarn run build",
+    "test": "jest --passWithNoTests"
+  },
+  "contributors": [
```

Comment:
Don't be shy to add here yourself as well.

Reply:
I don't think it makes a difference until we don't publish it. I'll consider it though.